helm-chart/templates/deployment.yaml

```diff
@@ -58,22 +58,19 @@ spec:
               subPath: comptest.yaml
             - name: staticfiles
               mountPath: /opt/staticfiles
+        {{- if .Values.adminUsers }}
         - name: admin-add
           image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           workingDir: /opt/frx-challenges/comptest
           command:
-          {{- if .Values.adminUsers }}
             - python
             - manage.py
             - promote
             {{- range .Values.adminUsers }}
             - {{ . }}
             {{- end }}
-          {{- else }}
-            - echo
-            - "No admin users to add"
-          {{- end }}
+        {{- end }}
```

Comment:
this should be after line 81, right?

Reply:
Yes, sorry, I closed the statement wrong